src/wallet/rpcwallet.cpp

```diff
@@ -132,7 +132,7 @@ UniValue getaddressinfo(const UniValue& params, bool fHelp)
 
     const std::string example_address = "\"DMJRSsuU9zfyrvxVaAEFQqK4MxZg6vgeS6\"";
 
-    if (fHelp || params.size() > 1)
+    if (request.fHelp || request.params.size() > 1)
         throw std::runtime_error(
                 "getaddressinfo ( \"address\" )\n"
                 "\nReturn information about the given PIVX address.\n"
```

Comment:
It was already like this, but shouldn't the address be a __required__ field? i.e.
```cpp
if (request.fHelp || request.params.size() != 1)
    throw std::runtime_error(
        "getaddressinfo \"address\"\n"
        ...
```